src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/AppointmentData.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.reporting.ndmis.performance
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Attended
+import java.time.OffsetDateTime
+import java.util.*
+
+data class AppointmentData(
+  val referralReference: String,
+  val referralId: UUID,
+  val appointmentId: UUID,
+  val appointmentTime: OffsetDateTime,
+  val durationInMinutes: Int,
+  val bookedAt: OffsetDateTime,
+  val attended: Attended?,
+  val attendanceSubmittedAt: OffsetDateTime?,
+  val notifyPPOfAttendanceBehaviour: Boolean?,
+  val deliusAppointmentId: String,
+  val reasonForAppointment: AppointmentReason // saa or delivery
+) {
+  companion object {
+    val fields = listOf(
+      "referralReference",
+      "referralId",
+      "appointmentId",
+      "appointmentTime",
+      "durationInMinutes",
+      "bookedAt",
+      "attended",
+      "attendanceSubmittedAt",
+      "notifyPPOfAttendanceBehaviour",
+      "deliusAppointmentId",
+      "reasonForAppointment",
+    )
+    val headers = listOf(
+      "referral_ref",
+      "referral_id",
+      "appointmentId",
+      "appointment_time",
+      "duration_in_minutes",
+      "booked_at",
+      "attended",
+      "attendance_submitted_at",
+      "notifyppof_attendance_behaviour",
+      "delius_appointment_id",
+      "reason_for_appointment",
+    )
+  }
+}
+
+enum class AppointmentReason {
+  SAA,
+  DELIVERY;
+
+//  override fun toString(): String {
```

Comment:
you can also explicitly add the values to the constructor here instead of lowercasing the name

```
enum class Reason(
 val value: String,
) {
	SAA("saa"),
    DELIVERY("delivery");

	override fun toString(): String {
        return value
    }
}
```

either is fine